### DISCUSSION TITLE
Information updated every 30 minutes at night while plugged in

### DISCUSSION THREAD

**Discussion by nis** [Hyundai-Kia-Connect/kia_uvo#754] [Q&A]
My 2020 Hyundai Kona eats 12V batteries. Winter is coming here in the north (Denmark) and more often than not, I need to boost the car to get it started. Which is annoying.

I noticed that it seems like the information from the car is updated every 30 minutes at night. Look at the left side of the attached image. Last night the battery ran out in the evening, so no updates tonight.

I have both scan intervals in the integration set to 240 minutes and "No Force Refresh Start Hour" set to 23 and the other end to 5. So I believe that the integration is not what triggers these updates.

Is the 30 minute intervals at night, when hooked up to the charger, a pattern you are seeing as well?

I have had other apps using my Bluelink account but I believe I have removed all of them. (Is there a way to see if there is any I have missed?) 

![Update History](https://github.com/Hyundai-Kia-Connect/kia_uvo/assets/2558/dcc2e232-4b85-4b9f-a28e-3a0ff69ec511)

**Comment by cdnninja** [comment]
Could we get debug logs? And a screenshot of your interval config.

**Comment by fuatakgun** [comment]
Car is sending its information to the cloud without any intervention, for my PHEV, the cached information is updated in every 10% of charge of battery, even at night.

This is how KIA/Hyundai app is also functioning, relying on regular updates from car.

**Comment by nis** [comment]
I decided to move my HA install from a HA Core to a Proxmox VM with HA OS. So it's hard to say what exactly did the trick. But now the update of the information is updated at the intervals i set.
